README.md

```diff
@@ -330,753 +277,454 @@ Rails.application.routes.draw do
-
-This updating will be very similar to creating new tasks. At a high level, we will need to:
-
-- Create a button to edit a specific task
-- Create a route that will GET us an edit form
-  - Add the action and views for this route
-- Create a route that will update a specific task based on form input
-  - Add the action that will accomplish this update and redirect the user
-
-### Add an Edit Button to the Show Page
-
-To make our user experience as smooth as possible, let's create a link to 'Edit' a specific task - similar to the link we created to add a new task.
-
-In our `show.html.erb` file, update the code to include an edit link:
-
-```html
-<a href="/tasks">Task Index</a>
+There's a lot going on here - let's break it down.
 
-<h1><%= @task.title %></h1>
+Before we can make any updates to a record in our database, we first need to find that information, which is why we are using the find method that we used on our show action. Then, we can use an ActiveRecord update to change the object that ActiveRecord found and created for us. Finally, we need to save the changes we made to that object in our database. And, thinking way back to when we were creating an object, we want to return the full object to the user as json.
 
-<p><%= @task.description %></p>
+Now that we have this edit functionality implemented, make sure you have your server running, and play around with sending requests in postman to update tasks.
 
-<a href="/tasks/<%= @task.id %>/edit">Edit</a>
-```
+## Deleting a Task
 
-Now, when a user navigates to [http://localhost:3000/tasks/1](http://localhost:3000/tasks/1), they will see a link to 'Edit'. If they click on that link, a `GET` request will be sent to the URI `tasks/:id/edit`. As of now, that route doesn't exist - let's create it!
+As of now, we have the Create, Read, and Update functions done - all we are missing is Delete!
 
-### Creating the Route, Controller Action, and View
+### Add a Delete Route and Controller Action
 
-In our `config/routes.rb` file, add the following route:
+Let's go add that route to our `config/routes.rb`:
 
 **config/routes.rb**
-
 ```ruby
-get '/tasks/:id/edit', to: 'tasks#edit'
+delete '/tasks/:id', to: 'tasks#destroy'
 ```
 
-And in our tasks controller, add the following action:
-
-**app/controllers/tasks_controller.rb**
+And, with this route, we will need to add a destroy action to our tasks controller:
 
 ```ruby
-def edit
-  @task = Task.find(params[:id])
+def destroy
+    render json: Task.delete(params[:id])
+    # TODO: what is the difference between delete and destroy? We do delete in the api tutorial and distroy in the old task manager.
+    # I think that delete returns 200 and no respose body, and destroy returns the deleted object, which is weird.
+    # I don't like that if you call delete on an id that doesn't exist you still get 200 ok and not an error...
```

Comment:
I believe this is the way that it is because the HTTP verb is DELETE but for whatever reason, Rails prefers `destroy` so it does get a little confusing. I always use `destroy` except when defining the http verb. I am very open to being corrected on this way of thinking. We can also override the default response, which I usually do. It would be a `204` which is no content.

As for deleting a record that doesn't exist and getting a 200, yeah that's odd. I usually have some kind of find error handling in place for that so I hadn't noticed. We could do that here. If we don't find the object return a 404 or something?

Reply:
I think some say REST convention is 200 for delete no matter what, even if that record never existed

Reply:
I'm just going to use the ActiveRecord delete to match the other endpoints.